### PR TITLE
Set Ulimits when creating docker container

### PR DESCRIPTION
**Pull request by siggisim**
---

**Version bump**: Patch

### CODE REVIEW

**Review by tylerwilliams — APPROVED** [review]
LGTM

File: enterprise/server/remote_execution/containers/docker/docker.go [hunk 1]
```diff
@@ -25,12 +25,14 @@ import (
 	dockertypes "github.com/docker/docker/api/types"
 	dockercontainer "github.com/docker/docker/api/types/container"
 	dockerclient "github.com/docker/docker/client"
+	units "github.com/docker/go-units"
```
Comment (bduffany) [inline enterprise/server/remote_execution/containers/docker/docker.go:28]: nit: prefix import alias with `docker` (for consistency with above imports)

**Review by bduffany — APPROVED** [review]

File: enterprise/server/remote_execution/containers/docker/docker.go (hunk 1 above)
Comment (siggisim) [reply enterprise/server/remote_execution/containers/docker/docker.go:28]: this library is called `go-units` it just happens to be in the docker organization: https://github.com/docker/go-units

Comment (bduffany) [reply enterprise/server/remote_execution/containers/docker/docker.go:28]: woops -- units sg

**Pull request merged**